spec/features/public/visitors/javascript_detection_spec.rb

```diff
@@ -1,15 +0,0 @@
-RSpec.feature "JavaScript detection" do
```

Comment:
If we're removing the test do we need to have the `js-enabled`  class remain in the layout, or is that used as a conventional way to communicate with other tooling such as screen readers?

Reply:
My view is that it is part of the GOVUK frontend template so we should keep it, but happy either way:

https://design-system.service.gov.uk/styles/page-template/